node/actors/network/src/config.rs

```diff
@@ -42,6 +44,10 @@ impl Default for RpcConfig {
                 burst: 10,
                 refresh: time::Duration::ZERO,
             },
+            push_l1_batch_signature_rate: limiter::Rate {
+                burst: 10,
```

Comment:
this should be low. Let's start with burst = 2, and refresh = 500ms. In tests we can use a higher/infinite rate limit of course.